src/lex_utils.cc

```diff
@@ -44,11 +44,16 @@ lsPosition CharPos(const std::string& search,
   return result;
 }
 
-bool ShouldRunIncludeCompletion(const std::string& line) {
-  size_t start = 0;
+bool ShouldRunIncludeCompletion(const std::string& line, size_t& start) {
```

Comment:
I like to use include completion by typing "#foo.cc", maybe we can add a config parameter if we should default to `"` or `<` in this case. Changing to let the user pick `<` or `"` by typing `#include <` otherwise sgtm since I think the existing algorithm doesn't work reliably enough.